_content/monographs/karma-vibhangam_anandajoti.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Karma-vibhaṅgam"
+editor: "Sylvain Lévi"
+translator: anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Karma-vibhanga/Karma-vibhanga.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1N8mDGdmZ34DYpWuhcqN3cmv0pLGN6mgy/view?usp=sharing"
+course: rebirth-stories
+tags:
+  - ethics
+  - karma
+year: 2020
+month: apr
+pages: 180
+---
+
+This is Ānandajoti Bhikkhu's translation of Sylvain Lévi's monumentous French work on the Karma-vibhaṅga (Analysis of Deeds). This text, found in the vibhaṅga section of the Abhidhamma Pitaka in the Pāli Canon, focuses on listing good and bad deeds along with their results. The first 14 deeds are also found in the Cūḷa-kamma-vibhaṅga-sutta of the Pāḷi Majjhima-nikāya (MN 135), while the remaining deeds are unique to this text. This teaching from the Buddha arises after an interesting interaction during his alm's round.
```

Comment:
This text is in the canon, so it should be in the canon folder (not monographs) and the name should be `mkv.md` to match the SuttaCentral id for the text